game/scripts/npc/abilities/legion_commander_duel.txt

```diff
@@ -55,7 +55,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "reward_damage"                                   "10 18 26 42 58"
+        "reward_damage"                                   "12 20 28 44 60" //OAA
```

Comment:
I just noticed that its +2 dmg at all levels. So meaningless ...

Reply:
I reverted the cooldown reduction and set the bonus damage to "12 24 36 48 60". I still think this hero will need the reduced cooldown ultimate for the reasons I stated, but I am happy to see how these changes play out. 

Some other LC obversations. (I wrote this because I think alot of people think LC is better then she actually is because infinite damage lul). 
-LC is exceptionally easy to counter in OAA with just item purchases (linkens, full tank or eblade). People can easily get tankier faster then you get duel wins (Aura items amplify this issue). 
-Most heroes which get hard countered by strong dispels are bad, this is a large reason why you would pick LC. 
-LC doesn't really excell in teamfights, she is more of a pickoff hero. OAA is basically 5v5 teamfights after 10 minutes. LC doesn't mesh well into this playstyle.)
- There are alot of heroes which just counter LC (especially supports). These are OAA supports so they can have any item to prevent you from killing them with duel. 
- LC really struggles for items. Blademail duration is insanely nerfed and is only good on Axe (It doesn't get its dota duration until level 4. Lol?). If you go full damage so you can kill people in duels you have no HP. If you go a balanced build you don't have enough damage to kill people.